apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryPage.cs

```diff
@@ -31,7 +31,8 @@ public sealed class BigQueryPage
         public JobReference JobReference { get; }
 
         /// <summary>
-        /// Reference to the table this result set was fetched from.
+        /// Reference to the table this result set may has been fetched from.
+        /// May be null, for instance, script queries don't store results in tables.
```

Comment:
I'd replace this sentence with "May be null. (For example, script queries don't store results in tables.)" I think that's just a bit clearer.
(Ditto all the other occurrences.)

Reply:
Done, here and elsewhere.